tests/select-ranges.test.ts

```diff
@@ -81,39 +82,42 @@ describe('live demo', () => {
       await sidebar.clickButton('EXTRACT TEXT');
       await sidebar.switchToAppIFrame();
 
-      await driver.wait(driver.findElement(MARKING_LOCATOR));
+      await driver.wait(driver.findElement(By.css("." + MARKING_CSS_CLASS)));
       await screenShooter.shoot('after-extract-text');
     });
 
-    it('display the extracted text', async () => {
-      const markingsParent = await driver.findElement(MARKING_LOCATOR).findElement(By.xpath('..'));
-      expect(await markingsParent.getText()).toEqual(TEST_TEXT);
+    test('display the extracted text', async () => {
+      const markingsParent = await driver.findElement(By.css("." + MARKING_CSS_CLASS)).findElement(By.xpath('..'));
```

Comment:
I think we can omit the `"." +` here and in line 90 below too

Reply:
Used marking locator which is readily defined